credentials/alts/internal/authinfo/authinfo_test.go

```diff
@@ -43,6 +43,8 @@ const (
 	testLocalHostname  = "local_hostname"
 )
 
+var testPeerAttributes = map[string]string{}
```

Comment:
Don't use a package level variable; initiate the variable inside the test.

Reply:
Will do thank you